edb/schema/delta.py

```diff
@@ -1660,6 +1667,12 @@ def has_ddl_identity(self, aspect: str) -> bool:
             and self.ddl_identity.get(aspect) is not None
         )
 
+    def set_ddl_identity(self, aspect: str, value: Any) -> None:
+        if self.ddl_identity is None:
+            self.ddl_identity = {}
+
+        self.ddl_identity[aspect] = value
```

Comment:
Is it OK if we're overriding some aspect that's already there?  Or should there be separate API for that?

Reply:
Overwriting existing values is harmless here.